DATA/UNIVERSE/SYSTEMS/Br01/Br01.ini

```diff
@@ -10289,3 +10289,215 @@ parent = Br01_08
 visit = 128
 Archetype = jump_hole_invisible
 goto = Br01, br01_from_start_unlawful, gate_tunnel_bretonia
+
+;;Mineable Gas Pockets
+
+[Object]
+nickname = mineable_small_co2_Br01_01
+ids_name = 459177
+;res str
+; CO2 Gas Pocket

```

Comment:
The resource string parser allows to reuse the very same text and name again and again without having to repeatingly typing it into the files. Check out how it was done for the tradelane rings to see it.